src/App.jsx

```diff
@@ -13,34 +17,76 @@ export const goodsFromServer = [
   'Jam',
   'Garlic',
 ];
+export const App = () => {
+  const [sortType, setSortType] = useState(SORT_BY_FIELDS.none);
+  const [isReversed, setIsReversed] = useState(false);
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+  const resetGoods = () => {
+    setSortType(SORT_BY_FIELDS.none);
+    setIsReversed(false);
+  };
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const visibleGoods = getVisibleGoods(
+    goodsFromServer,
+    sortType,
+    isReversed,
+    SORT_BY_FIELDS,
+  );
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  return (
+    <div className="section content">
+      <div className="buttons">
+        <button
+          onClick={() => setSortType(SORT_BY_FIELDS.alphabetic)}
+          type="button"
+          className={cn('button', {
+            'is-info': sortType === SORT_BY_FIELDS.alphabetic,
+            'is-light': sortType !== SORT_BY_FIELDS.alphabetic,
```

Comment:
The `is-light` class should be removed from the 'Sort alphabetically' button when this order is applied, as per the task description. Consider adjusting the logic to ensure this class is not present when the button is active.